dev/preview/previewctl/pkg/preview/preview.go

```diff
@@ -55,18 +60,21 @@ func New(branch string, logger *logrus.Logger) (*Preview, error) {
 	}
 
 	return &Preview{
-		branch:     branch,
-		namespace:  fmt.Sprintf("preview-%s", GetName(branch)),
-		name:       GetName(branch),
-		kubeClient: harvesterConfig,
-		logger:     logEntry,
+		branch:          branch,
+		namespace:       fmt.Sprintf("preview-%s", GetName(branch)),
+		name:            GetName(branch),
+		kubeClient:      harvesterConfig,
+		logger:          logEntry,
+		vmiCreationTime: nil,
 	}, nil
 }
 
-func (p *Preview) InstallContext(watch bool, timeout time.Duration) error {
+func (p *Preview) InstallContext(wait bool, timeout time.Duration) error {
```

Comment:
the flag is called `watch` right? Any reason to rename it to `wait`?

Reply:
I renamed it to better reflect what it meant for the function. The current implementation would just wait for a single VM to become ready and then install the context. It didn't "watch" for changes to the VM after or changes to the branch after the first initial success.

I will have a look and see if I can extend this function to do that watching as you suggest in your PR comment - if I can it makes sense to keep the name as "watch" instead of "wait" ☺️